Controller/Adminhtml/Import/Ajax.php

```diff
@@ -81,12 +81,11 @@ public function execute()
             $importType        = (string)$this->request->getParam('importType');
             $client            = $this->apiClient->getClient($storeId);
 
-            if ($chunkId == 0 && $importType == 'customers') {
-                $this->activityHelper->createActivity($storeId, 'import_start');
-            }
-
             switch ($importType) {
                 case 'customers':
+                    if ($chunkId == 0 && $importType == 'customers') {
```

Comment:
why do you need the `importType` check here?

Reply:
This activity needs to fire when import start and when it's done. I am using the import type to determine on witch import step to fire the activity event. In this case if the chunkId is 0 and the import type is customers this means the import process is started and import_start event should be fired.

Reply:
But you're already in the `customers` case statement?! Why do you need this check again?